crates/stages/stages/src/stages/tx_lookup.rs

```diff
@@ -147,16 +148,19 @@ where
                     .cursor_write::<tables::RawTable<tables::TransactionHashNumbers>>()?;
 
                 let total_hashes = hash_collector.len();
-                let interval = (total_hashes / 10).max(1);
+                let log_interval = Duration::from_secs(5);
```

Comment:
I'd like to make this a constant as well

Reply:
good catch, I'll reuse the LOG_INTERVAL defined in utils.rs